lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -69,6 +75,8 @@
 
 public class BuildCriOauthRequestHandler
         implements RequestHandler<JourneyRequest, Map<String, Object>> {
+    private static final List<Gpg45Profile> ACCEPTED_PROFILES =
+            List.of(Gpg45Profile.M1A, Gpg45Profile.M1B);
```

Comment:
Feels like this should be stored somewhere shared - we're repeating it 3 times now.

I'm not sure exactly where though - it's nice that the GPG45 Profile Evaluator is agnostic to the profiles that we currently support. Any ideas?

Reply:
Yeah noticed that but then thought will be done as part of introducing new GPG profile refactoring.

Reply:
Fair enough - I can make that a requirement of another ticket.

Reply:
done